nthuoj/settings.py

```diff
@@ -143,3 +143,11 @@
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
     'djangobower.finders.BowerFinder',
 )
+
+#maximum of public users for a single contest
+MAX_PUBLIC_USER = 200
+#public user username prefix
+PUBLIC_USER_PREFIX = "TEAM"
+
+PUBLIC_USER_DEFAULT_PASSWORD = "000"
```

Comment:
I will fix that in future PR.
Since I need to generate a csv for user's password,
I need to modify download function in views.py.
I think it will be better to separate them